src/wallets/hardware/ledger/index.js

```diff
@@ -145,10 +147,38 @@ const isWebUsbSupported = async () => {
   );
 };
 
+const isWebAuthnSupported = async () => {
+  // eslint-disable-next-line
+  const isSupported = !(typeof PublicKeyCredential == 'undefined');
+  return (
+    isSupported && platform.os.family !== 'Windows' // take it out later once the windows issue is fixed
+  );
+};
+
+const isWebHidSupported = async () => {
+  // WebUSB is currently only supported on Google Chrome / Chromium DEV version and by explicitly enabling chrome://flags/#enable-experimental-web-platform-features
+  const isSupported = await TransportWebHID.isSupported();
+  return (
+    isSupported && platform.os.family !== 'Windows' // take it out later once the windows issue is fixed
+  );
+};
+
 const getLedgerTransport = async () => {
   let transport;
   const support = await isWebUsbSupported();
-  if (support) {
+  // eslint-disable-next-line
+  console.log(support); // todo remove dev item
+  const webAuthnSupport = await isWebAuthnSupported();
+  // eslint-disable-next-line
+  console.log(webAuthnSupport); // todo remove dev item
+  const webHidSupport = await isWebHidSupported();
+  // eslint-disable-next-line
+  console.log(webHidSupport); // todo remove dev item
```

Comment:
console logs,
also how can I test this?

Reply:
You have to use a windows machine for it.